app/models/task.py

```diff
@@ -1,5 +1,22 @@
 from app import db
-
+from datetime import datetime 
 
 class Task(db.Model):
-    task_id = db.Column(db.Integer, primary_key=True)
+    task_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String) 
+    completed_at = db.Column(db.DateTime, nullable=True)
```

Comment:
The default value for `nullable` is `True`, so we don't need to list it explicitly.